tool_commands/commands/Step 1 - Check data/schematisation_checker.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+
+from sqlalchemy.exc import OperationalError
+from threedi_modelchecker import errors
+from threedi_modelchecker.exporters import format_check_results
+from threedi_modelchecker.model_checks import ThreediModelChecker
+
+from ThreeDiToolbox.tool_commands.base.custom_command import CustomCommandBase
+from ThreeDiToolbox.tool_commands.schematisation_checker.controller import \
+    SchemaCheckerDialogWidget
+from ThreeDiToolbox.utils.user_messages import pop_up_info
+from ThreeDiToolbox.utils.user_messages import progress_bar
+
+logger = logging.getLogger(__name__)
+
+
+class CustomCommand(CustomCommandBase):
+
+    def __init__(self, iface, ts_datasource, plugin_dir):
+        self.iface = iface
+        self.plugin_dir = plugin_dir
+
+    def show_gui(self):
+        """Show SchemaChecker dialog"""
+        self.modelchecker_widget = SchemaCheckerDialogWidget(
+            self.iface, command=self,
+        )
+        self.modelchecker_widget.exec_()
+
+    def run(self):
+        self.show_gui()
+
+    def run_it(self, threedi_db):
+        """Applies the threedi-modelchecker to `threedi_db`
+
+        The connection to the `threedi_db` and its south_migration_history are first
+        validated. Next, any model errors are written to a text file.
+        """
+        logger.info("Starting threedi-modelchecker")
+        try:
+            model_checker = ThreediModelChecker(threedi_db)
+            model_checker.db.check_connection()
+        except OperationalError as exc:
+            logger.exception("Failed to start a connection with the database.")
+            pop_up_info(
+                "Something went wrong trying to connect to the database, please check"
+                " the connection settings: %s" % exc.args[0]
+            )
+            return
+        except errors.MigrationMissingError:
+            logger.exception("The selected 3Di model does not have the latest "
+                             "migration")
+            pop_up_info(
+                "The selected 3Di model does not have the latest migration, please "
+                "migrate your model to the latest version."
+            )
+            return
+        except errors.MigrationTooHighError:
+            logger.exception(
+                "The selected 3Di model has a higher migration than expected."
+            )
+            pop_up_info(
+                "The 3Di model has a higher migration than expected, do you have "
+                "the latest version of ThreediToolbox?"
+            )
+            return
+        except errors.MigrationNameError:
+            logger.exception(
+                "Unexpected migration name, but migration id is matching. "
+                "We are gonna continue for now and hope for the best."
+            )
+
+        output_filename = 'Model_errors.txt'
```

Comment:
```suggestion
        output_filename = 'model-errors.txt'
```

Lowercase? En een streepje is voor windows net nog wat vriendelijker?
Smaakkwestie :-)